python/cuml/manifold/t_sne.pyx

```diff
@@ -427,30 +430,34 @@ class TSNE(Base):
             X contains a sample per row.
             Acceptable formats: cuDF DataFrame, NumPy ndarray, Numba device
             ndarray, cuda array interface compliant array like CuPy
+        convert_dtype : bool, optional (default = True)
+            When set to True, the fit_transform method will automatically
+            convert the inputs to np.float32.
 
         Returns
         --------
         X_new : array, shape (n_samples, n_components)
                 Embedding of the training data in low-dimensional space.
         """
-        self.fit(X)
+        self.fit(X, convert_dtype)
```

Comment:
As a paranoid individual, I usually do `self.fit(X, convert_dtype=convert_dtype)` since I see the order of non-obvious parameters get used wrong all the time.